### PR TITLE
Add Fedora Guide

### PR DESCRIPTION
**Pull request by nikodunk**

### CODE REVIEW

File: docs/distributions/fedora/installation.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Download the latest safe release
+
+Thanks so Mike for building. Download here. [https://github.com/mikeeq/mbp-fedora](https://github.com/mikeeq/mbp-fedora)
```
Comment (networkException) [inline docs/distributions/fedora/installation.md:3]: Maybe change this to "You can download a live iso here (link)"

File: docs/distributions/overview.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 This is a landing page for the distribution specific guides contained in this wiki.
 
-- [Arch Linux](https://wiki.t2linux.org/distributions/arch/installation/)
-- [Manjaro](https://wiki.t2linux.org/distributions/manjaro/home/)
-- [Ubuntu](https://wiki.t2linux.org/distributions/ubuntu/home/)
+-   [Arch Linux](https://wiki.t2linux.org/distributions/arch/installation/)
+-   [Manjaro](https://wiki.t2linux.org/distributions/manjaro/home/)
+-   [Ubuntu](https://wiki.t2linux.org/distributions/ubuntu/home/)
+-   [Fedora](https://wiki.t2linux.org/distributions/fedora/home/)
```
Comment (networkException) [inline docs/distributions/overview.md:8]: This list should stay in alphabetical order

**Review by networkException — CHANGES_REQUESTED** [review]

File: docs/distributions/fedora/installation.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+# Hardware Requirements
+
+-   USB-C to Ethernet adapter
+    -   This isn't required, and you can use a USB WiFi Adapter instead.
+-   USB-C to USB adapter
+
+# Install Procedure
+
+(These instructions are reused from the Manjaro installation guide)
```
Comment (Redecorating) [inline docs/distributions/fedora/installation.md:13]: In future we probably should consolidate these sections that are common to multiple distros (i.e. turn off secure boot, flash iso to usb) into one place and link to that from each distro's article, so we don't have to repeat ourselves and update more articles when things change. (But I don't think that needs to be done in this pr)

File: docs/distributions/fedora/installation.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+    3. rest of the free partition space or around 30GB to an ext4 partition mounted at /
+
+    4. (Optional) rest of the free partition space to an ext4 partition mounted at /home
+
+    5. Leave efi boot alone until using a [separate efi partition](https://wiki.t2linux.org/guides/windows/#using-seperate-efi-partitions).
+
+9. Continue the rest of the setup. If your Mac somehow turns off with the fans spinning at full speed, go to the various FAQs. If not, you should be fine.
+10. Once it's finished, you can reboot without your installation media. Hold down Option (⌥) while booting, then select EFI Boot and press enter.
+11. Welcome to Fedora! :)
```
Comment (Redecorating) [inline docs/distributions/fedora/installation.md:47]: Might want to prompt them to setup wifi and audio before the end. You could add a step either telling them to they can look at the fedora faq's page for audio and wifi, or linking directly to those two guides.

File: docs/distributions/fedora/installation.md (hunk 1 above)
Comment (nikodunk) [reply docs/distributions/fedora/installation.md:3]: done

File: docs/distributions/overview.md (hunk 2 above)
Comment (nikodunk) [reply docs/distributions/overview.md:8]: good point. done.

File: docs/distributions/fedora/faq.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Installing alongside Windows
+
+If you already have Bootcamp installed, you might notice that the boot option for Bootcamp instead boots you into Ubuntu. This is because GRUB automatically shares with a Windows installation. Follow [this guide on triple booting](https://wiki.t2linux.org/guides/windows/#if-windows-is-installed-first) to get Windows working again.
```
[the remark this answers is not in the document]
Comment (nikodunk) [reply docs/distributions/fedora/faq.md:3]: ack. of course. Thanks!

File: docs/distributions/fedora/installation.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+    2. (optional) 8GB swap partition (this can be larger depending on what you are using Ubuntu for)
+
+    3. rest of the free partition space or around 30GB to an ext4 partition mounted at /
+
+    4. (Optional) rest of the free partition space to an ext4 partition mounted at /home
+
+    5. Leave efi boot alone until using a [separate efi partition](https://wiki.t2linux.org/guides/windows/#using-seperate-efi-partitions).
+
+9. Continue the rest of the setup. If your Mac somehow turns off with the fans spinning at full speed, go to the various FAQs. If not, you should be fine.
```
[the remark this answers is not in the document]
Comment (nikodunk) [reply docs/distributions/fedora/installation.md:45]: Good idea. Done.

File: docs/distributions/fedora/installation.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+    1. (optional) a 1GB ext4 partiton mounted at /boot
+
+    2. (optional) 8GB swap partition (this can be larger depending on what you are using Ubuntu for)
+
+    3. rest of the free partition space or around 30GB to an ext4 partition mounted at /
+
+    4. (Optional) rest of the free partition space to an ext4 partition mounted at /home
+
+    5. Leave efi boot alone until using a [separate efi partition](https://wiki.t2linux.org/guides/windows/#using-seperate-efi-partitions).
```
[the remark this answers is not in the document]
Comment (nikodunk) [reply docs/distributions/fedora/installation.md:43]: Hmm definitely _unless_, though then that also doesn't make sense in the Ubuntu guide this was forked from here at https://github.com/t2linux/wiki/blob/master/docs/distributions/ubuntu/installation.md?
@EmreTech FYI

File: docs/distributions/fedora/installation.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+4. After disabling Secure Boot, hold down Option (⌥) while the Mac is rebooting.
+5. Plug in your USB Flash Drive/SD Card. If two Yellow EFI Boot options appear, select the one towards the very right and hit enter.
+6. You should now be in the GRUB boot menu. Select "Try Fedora Live CD".
+7. Once booted into Fedora, you can install it like normal until you get to the partition option. If you don't get to the graphic OS, try another USB stick. They are not all created alike.
+8. Find the partition you made before. MAKE SURE TO SELECT THE RIGHT PARTITION OR ELSE YOUR DATA WILL BE LOST. Delete it to make free space. You'll need to make these partitions:
+
+    1. (optional) a 1GB ext4 partiton mounted at /boot
+
+    2. (optional) 8GB swap partition (this can be larger depending on what you are using Ubuntu for)
```
[the remark this answers is not in the document]
Comment (nikodunk) [reply docs/distributions/fedora/installation.md:37]: Sorry yes. Did a proper search now.

File: docs/distributions/fedora/installation.md (hunk 3 above)
Comment (nikodunk) [reply docs/distributions/fedora/installation.md:13]: Good call.

File: docs/distributions/fedora/installation.md (hunk 4 above)
Comment (nikodunk) [reply docs/distributions/fedora/installation.md:47]: Good call.

Comment (nikodunk) [reply docs/distributions/fedora/installation.md:47]: Done.

File: docs/distributions/fedora/installation.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+Note before you begin: Due to Fedora having switched to using PipeWire, getting audio to work may be more difficult than on other distributions. See also [state](https://wiki.t2linux.org/state/). Confirmed working with headphones according to the [audio](https://wiki.t2linux.org/guides/audio-config/) guide, but speakers TBD. WiFi works as expected after following the [Wifi](https://wiki.t2linux.org/guides/wifi/) guide. Generally very usable and quick.
+
+# Download the latest safe release
+
+Thanks so Mike for building. You can download a live iso [here](https://github.com/mikeeq/mbp-fedora)
```
Comment (networkException) [inline docs/distributions/fedora/installation.md:5]: "Thanks so" doesn't seem right, maybe change this to "Thanks to"

Comment (nikodunk) [reply docs/distributions/fedora/installation.md]: Good catch. Done

**Review by Redecorating — APPROVED** [review]

**Review by networkException — APPROVED** [review]

**Pull request merged**